### PR TITLE
prime-run: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

Just as a note, although this is installable through the arch package manager, it is actually (to my knowledge) just a bash script

```bash
#!/bin/bash
__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia "$@"
```

### CODE REVIEW

File: pages/linux/prime-run.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# prime-run
+
+> Run a program using alternative graphics.
```
Comment (cyqsimon) [inline pages/linux/prime-run.md:3]:
```suggestion
> Run a program using alternative Nvidia graphics.
```

Comment (cyqsimon) [reply pages/linux/prime-run.md:3]: Otherwise LGTM.

**Review by cyqsimon — APPROVED** [review]

File: pages/linux/prime-run.md (hunk 1 above)
Comment (cyqsimon) [reply pages/linux/prime-run.md:3]: Otherwise LGTM.

File: pages/linux/prime-run.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# prime-run
+
+> Run a program using alternative Nvidia graphics.
+> More information: <https://wiki.archlinux.org/title/PRIME#PRIME_render_offload>.
+
+- Run a program using an Nvidia dedicated GPU:
```
Comment (kbdharun) [inline pages/linux/prime-run.md:6]:
```suggestion
- Run a program using a dedicated Nvidia GPU:
```

File: pages/linux/prime-run.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# prime-run
+
+> Run a program using alternative Nvidia graphics.
```
Comment (kbdharun) [inline pages/linux/prime-run.md:3]:
```suggestion
> Run a program using an alternative Nvidia graphics card.
```
How about this?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**